integrations/intercom/src/index.ts

```diff
@@ -346,7 +350,7 @@ async function sendMessage(
     attachmentUrls,
   })
 
-  await ack({ tags: { id: `${conversationParts.at(-1)?.id ?? ''}` } })
+  await ack({ tags: { id: conversationParts.at(-1)?.id ?? '' } })
```

Comment:
**nitpick:** this is still very confusing code; feel free to extract an extra variable next time